src/orquestra/sdk/__init__.py

```diff
@@ -6,10 +6,12 @@
 from . import mlflow, secrets
 from ._base._api import (
     CurrentRunIDs,
+    CurrentWorkflowIDs,
```

Comment:
**nitpick**: the word "workflow" is the central entity in our system. In the past we struggled with the ambiguity of "workflow IDs" sometimes meaning the wf def ID, sometimes wf run ID. `project_id`, `workspace_id`, and `config_name` are much more Platform-related than workflow graph-related. What would you say about this instead?

```suggestion
    CurrentExecutionIDs,
```